# HG changeset patch
# User Francois Bissey <f.r.bissey@massey.ac.nz>
# Date 1256518558 -46800
# Node ID 5545e2dbcad89e407608a8f305e2fdd3ea215e77
# Parent  a563085a78ce44c71a511c005e6566595ab67680
Correcting typo LCOAL to LOCAL in spkg-install

diff -r a563085a78ce -r 5545e2dbcad8 spkg-install
--- a/spkg-install	Mon Jan 19 02:48:26 2009 -0800
+++ b/spkg-install	Mon Oct 26 13:55:58 2009 +1300
@@ -19,7 +19,7 @@
 # remove old executable/headers/library
 echo "Removing old binary, headers and static library"
 rm -f "$SAGE_LOCAL"/bin/ecm
-rm -f "$SAGE_LCOAL"/lib/libecm.*
+rm -f "$SAGE_LOCAL"/lib/libecm.*
 rm -r "$SAGE_LOCAL"/include/ecm.h
 
 cd src
